project.yml

```diff
@@ -0,0 +1,157 @@
+name: SwifterSwift
+options:
+  generateEmptyDirectories: true
+fileGroups:
+  - Examples
+configFiles:
+  Debug: Configs/Project-Debug.xcconfig
+  Release: Configs/Project-Release.xcconfig
+targets:
+  SwifterSwift-iOS:
+    type: framework
+    platform: [iOS]
+    platformSuffix: ""
+    scheme:
+      testTargets:
+        - SwifterSwift-iOSTests
+    configFiles:
+      Debug: Configs/SwifterSwift-iOS-Debug.xcconfig
+      Release: Configs/SwifterSwift-iOS-Release.xcconfig
+    settings:
+      PRODUCT_NAME: SwifterSwift
+    sources:
+      - path: Sources
+        excludes:
+          - "SwifterSwift/AppKit/*"
+  SwifterSwift-tvOS:
+    type: framework
+    platform: [tvOS]
+    platformSuffix: ""
+    scheme:
+      testTargets:
+        - SwifterSwift-tvOSTests
+    configFiles:
+      Debug: Configs/SwifterSwift-tvOS-Debug.xcconfig
+      Release: Configs/SwifterSwift-tvOS-Release.xcconfig
+    settings:
+      PRODUCT_NAME: SwifterSwift
+    sources:
+      - path: Sources
+        excludes:
+          - "SwifterSwift/UIKit/UIGestureRecognizerExtensions.swift"
+          - "SwifterSwift/UIKit/UISearchBarExtensions.swift"
+          - "SwifterSwift/UIKit/UIDatePickerExtensions.swift"
+          - "SwifterSwift/UIKit/UIStackViewExtensions.swift"
+          - "SwifterSwift/UIKit/UIScrollViewExtensions.swift"
+          - "SwifterSwift/UIKit/UIWindowExtensions.swift"
+          - "SwifterSwift/UIKit/UIRefreshControlExtensions.swift"
+          - "SwifterSwift/UIKit/UIActivityExtensions.swift"
+          - "SwifterSwift/CoreAnimation/CAGradientLayerExtensions.swift"
+          - "SwifterSwift/AppKit/*"
+  SwifterSwift-watchOS:
+    type: framework
+    platform: [watchOS]
+    platformSuffix: ""
+    configFiles:
+      Debug: Configs/SwifterSwift-watchOS-Debug.xcconfig
```

Comment:
As much as I love seperating configs into their own files, I don't think we need them for this project
Must of the configs we have are Xcode's defaults